fbchat/client.py

```diff
@@ -135,30 +130,24 @@ def login(self):
             self.start_time = now()
             self.uid = int(self._session.cookies['c_user'])
             self.user_channel = "p_" + str(self.uid)
-            self.ttstamp = ''
 
             r = self._get(BaseURL)
             soup = bs(r.text, "lxml")
-            self.fb_dtsg = soup.find("input", {'name':'fb_dtsg'})['value']
```

Comment:
No need to make `fb_dtsg` a global variable, since it is only used in this function.